test/compare_errors.jl

```diff
@@ -0,0 +1,33 @@
+include("old_engine/NurseScheduling.jl")
```

Comment:
Prosiłbym o przetestowanie na wszystkich schedulach jakie mamy, to właśnie jest przewaga testów automatycznych.

Reply:
Linkuje mój kod sprzed niedawna, zrób więcej testów, ponieważ to jest bardziej poważna aplikacja, a errory są kluczowe.

https://github.com/prenc/evolutionary-regex-miner/blob/master/test/evolution.jl